packages/block-editor/src/components/block-list/use-block-props/use-focus-first-element.js

```diff
@@ -28,11 +28,16 @@ import { store as blockEditorStore } from '../../../store';
  */
 export function useFocusFirstElement( { clientId, initialPosition } ) {
 	const ref = useRef();
-	const { isBlockSelected, isMultiSelecting } = useSelect( blockEditorStore );
+	const { isBlockSelected, isMultiSelecting, __unstableGetEditorMode } =
+		useSelect( blockEditorStore );
 
 	useEffect( () => {
 		// Check if the block is still selected at the time this effect runs.
-		if ( ! isBlockSelected( clientId ) || isMultiSelecting() ) {
+		if (
+			! isBlockSelected( clientId ) ||
+			isMultiSelecting() ||
+			__unstableGetEditorMode() === 'zoom-out'
```

Comment:
Do we sill need these changes? In my testing it works fine without them.

Reply:
Unfortunately, yes. Maybe there's a better way though. If you remove that line, it breaks when:

- Place cursor in a paragraph (or some block you can type in)
- Click zoom out button
- Focus is not on zoom out button